### ISSUE TITLE
crash predicting for a configuration with all atoms outside of cutoff

### ISSUE THREAD

**Issue by bernstei** [ACEsuit/mace#31]
Trying to predict energy/forces/virial on a configuration with two atoms that are farther than the cutoff crashes (see below).  I'm guessing that the code that fills zeros for the gradient needs to take the true shape of the positions array into account?

```
Traceback (most recent call last):
  File "/home/cluster2/bernstei/src/work/GeSbTe/GAP_ACE_MACE/./evaluate_potentials.py", line 86, in <module>
    plot_dimers(calc_use, dimer_configs, pot_file_name)
  File "/home/cluster2/bernstei/src/work/GeSbTe/GAP_ACE_MACE/./evaluate_potentials.py", line 52, in plot_dimers
    Es.append(at.get_potential_energy())
  File "/home/Software/python/system/extra/lib/python3.9/site-packages/ase/atoms.py", line 730, in get_potential_energy
    energy = self._calc.get_potential_energy(self)
  File "/home/Software/python/system/extra/lib/python3.9/site-packages/ase/calculators/abc.py", line 24, in get_potential_energy
    return self.get_property(name, atoms)
  File "/home/Software/python/system/extra/lib/python3.9/site-packages/ase/calculators/calculator.py", line 501, in get_property
    self.calculate(atoms, [name], system_changes)
  File "/home/cluster2/bernstei/.local/lib/python3.9/site-packages/mace/calculators/mace.py", line 70, in calculate
    out = self.model(batch, compute_stress=True)
  File "/home/Software/python/system/extra/lib64/python3.9/site-packages/torch/nn/modules/module.py", line 1130, in _call_impl
    return forward_call(*input, **kwargs)
  File "/home/cluster2/bernstei/.local/lib/python3.9/site-packages/mace/modules/models.py", line 304, in forward
    forces, virials, stress = get_outputs(
  File "/home/cluster2/bernstei/.local/lib/python3.9/site-packages/mace/modules/utils.py", line 135, in get_outputs
    forces, virials, stress = compute_forces_virials(
  File "/home/cluster2/bernstei/.local/lib/python3.9/site-packages/mace/modules/utils.py", line 72, in compute_forces_virials
    torch.zeros_like(positions).expand(1, 1, 3),
RuntimeError: The expanded size of the tensor (1) must match the existing size (2) at non-singleton dimension 1.  Target sizes: [1, 1, 3].  Tensor sizes: [2, 3]
```

**Comment by bernstei** [comment]
Is it just a matter of replacing the `.expand(1, 1, 3)` with `.expand(-1, 1, 3)` ?

**Comment by bernstei** [comment]
Or, I see from other virials, just `torch.zeros((1,3,3))` ?

**Comment by bernstei** [comment]
Addressed by #26 in 1a04b2ee8457

**Comment by ilyes319** [comment]
Merged #26, closing now.
